makefiles/testing.mak

```diff
@@ -144,22 +151,24 @@ clean-tmain:
 # Test installation
 #
 tinst:
-	@ \
-	\
+	$(V_RUN) \
 	builddir=$$(pwd); \
 	rm -rf $$builddir/$(TINST_ROOT); \
-	\
 	$(SHELL) $(srcdir)/misc/tinst $(srcdir) $$builddir/$(TINST_ROOT)
 
 #
 # Test readtags
 #
+if USE_READCMD
 roundtrip: $(READ_TEST)
-	@ \
-	\
+	$(V_RUN) \
 	builddir=$$(pwd); \
-	\
 	$(SHELL) $(srcdir)/misc/roundtrip $(READ_TEST) $${builddir}/Units
+else
+# apparently Automake doesn't like adding an extra dependency in a conditional,
+# so add it unconditionally and simply have it not do anything if it's disabled.
```

Comment:
Not sure why my comment didn't make it here, but anyway: this comment is kind of outdated and could go